src/executor/mod.rs

```diff
@@ -98,12 +98,12 @@ impl<D: Deliverable, C: 'static + Connect + Clone + Send + Sync> Executor<D, C>
         // See a relevant article: https://www.extrahop.com/company/blog/2016/tcp-nodelay-nagle-quickack-best-practices/
         http.set_nodelay(true);
         http.set_keepalive(Some(keep_alive_timeout));
+
         let connector = A::wrap(HttpsConnector::from((http, tls)));
 
         let client = Arc::new(
-            hyper::Client::builder()
-                .keep_alive(true)
-                .keep_alive_timeout(Some(keep_alive_timeout))
```

Comment:
These changes scare me a lot because it took forever to get hyper keep alive working properly. I'm guessing this has become the default when calling `pool_idle_timeout` which is like an indication of when to reap connections?

Reply:
Yes, the default is to us a connection pool, and the `keep_alive` and `keep_alive_timeout` methods were deprecated due to their misleading names. 

https://github.com/hyperium/hyper/blob/d1d2f32a7358c1c7d489ebbb98f4cbfdca9bb573/src/client/client.rs#L884

https://github.com/hyperium/hyper/blob/d1d2f32a7358c1c7d489ebbb98f4cbfdca9bb573/src/client/client.rs#L900